confidant/authnz/userauth.py

```diff
@@ -113,11 +113,12 @@ def set_expiration(self):
                 max_expiration = now + datetime.timedelta(seconds=max_lifetime)
                 session['max_expiration'] = max_expiration
 
-    def set_current_user(self, email, first_name=None, last_name=None):
+    def set_current_user(self, **kwargs):
```

Comment:
Prefer adding role=None to the kwargs, than using ``**kwargs``. This change as currently made makes all arguments required.

Reply:
fixed